bundles/org.openhab.binding.modbus.e3dc/README.md

```diff
@@ -0,0 +1,371 @@
+| Supported Registers   | info             | supported-registers | Number | Number of registers supported by Modbus           |
+| Manufacturer Name     | info             | manufacturer-name   | String | Name of the Device Manufacturer                   |
+| E3DC Model Name       | info             | model-name          | String | Name of the E3DC Model                            |
+| E3DC Firmware Release | info             | firmware-release    | String | Firmware installed on this particular E3DC Device |
+| E3DC Serial Number    | info             | serial-number       | String | Serial Number of this particular E3DC Device      |
+
+
+### Channel Group _Power Block_
+
+| Channel Label                 |  Channel Group ID | Channel ID                   | Type                   | Description                  |
+|-------------------------------|-------------------|------------------------------|------------------------|------------------------------|
+| PV Output                     | power             | pv-power-supply              |  Number:Power          | Photovoltaic Power Production    |
+| Battery Discharge             | power             | battery-power-supply         |  Number:Power          | Battery discharges and provides Power    |
+| Battery Charge                | power             | battery-power-consumption    |  Number:Power          | Battery charges and consumes Power    |
+| Household Consumption         | power             | household-power-consumption  |  Number:Power          | Household consuming Power    |
+| Grid Power Consumption        | power             | grid-power-consumption       |  Number:Power          | Grid Power is needed in order to satisfy your overall Power consumption    |
+| Grid Power Supply             | power             | grid-power-supply            |  Number:Power          | More Photovoltaic Power is produced than needed. Additional Power is provided towards the Grid    |
+| External Power Supply         | power             | external-power-supply        |  Number:Power          | Power produced by an external device which is attached to your E3DC device    |
+| Wallbox Power Consumption     | power             | wallbox-power-consumption    |  Number:Power          | Power consumption of attached Wallboxes    |
+| Wallbox PV Power Consumption  | power             | wallbox-pv-power-consumption |  Number:Power          | Photovoltaic Power consumption (PV plus Battery) of attached Wallboxes    |
+| Autarky                       | power             | autarky-channel              |  Number:Dimensionless  | Your current Autarky Level    |
+| Self Consumption              | power             | self-consumption             |  Number:Dimensionless  | Your current Photovoltaic Self Consumption Level    |
+| Battery State Of Charge       | power             | battery-soc                  |  Number:Dimensionless  | Charge Level of your attached Battery    |
```

Comment:
If these are percent, you could mention it in the description.

Reply:
Done in readme.md and channel-types.xml to be consistent